### ISSUE TITLE
back remove email priority

### ISSUE THREAD

**Issue by zonky2** [contao/contao#608]
Currently, the e-mail priority of extensions like the Notification Center can be set, but this is not passed on by the Contao core. The priority was removed in 2016.
* https://github.com/contao/core/issues/8298
* https://github.com/contao/core/commit/ee4b7f495a7207cb9e30ed7f156ddf0f375eb516

I can't see that the priority setting theme has a devaluation in the spam filters - According to PHPmailer, this should only be done if the priority is set to 3 (normal) https://github.com/PHPMailer/PHPMailer/issues/449

It would be nice if this could be re-implemented as follows

https://github.com/contao/contao/blob/master/core-bundle/src/Resources/contao/library/Contao/Email.php#L455-L459

```diff
        // Set the return path (see #5004)
        $this->objMessage->setReturnPath($this->strSender);
+
+        // Set priority
+        if ($this->priority !== 3) {
+            $this->objMessage->setPriority($this->priority);
+        }

        // Send the e-mail
        $intSent = $this->objMailer->send($this->objMessage, $this->arrFailures);

```
see https://github.com/terminal42/contao-notification_center/issues/185

**Comment by fritzmg** [comment]
I think it would be fine to add the priority setting back in and only set it if it's not the default.

**Comment by leofeyer** [comment]
IMHO, the whole mail priority concept is broken by design, as prioritizing mails needs to be done by the **recipient** and not by the sender. Clients tend to always mark their mails as "high priority" even though most of the time they are not. 😄 

So I vote against re-adding this.

**Comment by aschempp** [comment]
Then I think we should not support any mail functionality anyway, because it can be abused as spam 🤣

**Comment by leofeyer** [comment]
Well, it is similar to the "priority" option in ticket systems such as Trac or osTicket. Users cannot estimate what priority a ticket will have for the maintainers, therefore the field does not make sense. And that is probably also why you cannot find a "priority" field when creating issues on GitHub.

**Comment by aschempp** [comment]
But emails are not sent by users. They are likely sent by a CMS, where the administrator configures the priority. For example you could set the priority of an order configuration to hight for your own webshop.

**Comment by leofeyer** [comment]
As discussed in Mumble on August 29th, we want to set `protected $intPriority = null` and only add the priority if the `setPriority()` method has been called (which Contao itself must never do!).

**Comment by leofeyer** [comment]
Fixed in d19f747865194b6b101a986ca56ecaa369bcd82e.
